src/test/java/net/sf/jabref/openoffice/CitationManagerTest.java

```diff
@@ -0,0 +1,77 @@
+package net.sf.jabref.openoffice;
+
+import static org.junit.Assert.*;
+
+import java.util.Optional;
+
+import org.junit.Test;
+
+import net.sf.jabref.openoffice.CitationManager.CitEntry;
+
+public class CitationManagerTest {
+
+    @Test
+    public void testCitEntryInitalPageInfo() {
+        CitEntry citEntry = new CitEntry("RefMark", "Context", Optional.of("Info"));
+        assertFalse(citEntry.pageInfoChanged());
+        assertTrue(citEntry.getPageInfo().isPresent());
+        assertEquals("Info", citEntry.getPageInfo().get());
+        assertEquals("RefMark", citEntry.getRefMarkName());
+        assertEquals("Context", citEntry.getContext());
+    }
+
+    @Test
+    public void testCitEntryInitalPageInfoChanged() {
+        CitEntry citEntry = new CitEntry("RefMark", "Context", Optional.of("Info"));
+        citEntry.setPageInfo(Optional.of("Other info"));
+        assertTrue(citEntry.pageInfoChanged());
+        assertTrue(citEntry.getPageInfo().isPresent());
+        assertEquals("Other info", citEntry.getPageInfo().get());
+    }
+
+    @Test
+    public void testCitEntryInitalPageInfoRemoved() {
+        CitEntry citEntry = new CitEntry("RefMark", "Context", Optional.of("Info"));
+        citEntry.setPageInfo(Optional.empty());
+        assertTrue(citEntry.pageInfoChanged());
+        assertFalse(citEntry.getPageInfo().isPresent());
+    }
+
+    @Test
+    public void testCitEntryNoInitalPageInfo() {
+        CitEntry citEntry = new CitEntry("RefMark", "Context", Optional.empty());
+        assertFalse(citEntry.pageInfoChanged());
+        assertFalse(citEntry.getPageInfo().isPresent());
+    }
+
+    @Test
+    public void testCitEntryNoInitalPageInfoChanged() {
+        CitEntry citEntry = new CitEntry("RefMark", "Context", Optional.empty());
+        citEntry.setPageInfo(Optional.of("Other info"));
+        assertTrue(citEntry.pageInfoChanged());
+        assertTrue(citEntry.getPageInfo().isPresent());
+        assertEquals("Other info", citEntry.getPageInfo().get());
+    }
+
+    @Test
+    public void testCitEntryEquals() {
+        CitEntry citEntry1 = new CitEntry("RefMark", "Context", Optional.of("Info"));
+        CitEntry citEntry2 = new CitEntry("RefMark2", "Context", Optional.of("Info"));
+        CitEntry citEntry3 = new CitEntry("RefMark", "Other Context", Optional.of("Other Info"));
+        assertTrue(citEntry1.equals(citEntry1));
+        assertTrue(citEntry1.equals(citEntry3));
```

Comment:
Yeah, that I have no idea about... My guess is that it is used for sorting a table and then this is a quicker way (the refMarks should be separate anyway, although I'm not sure what happens if you create two with the same name...).